.circleci/getChangelogs.sh

```diff
@@ -21,27 +22,25 @@ function cloneRepository() {
     CHANGELOG_FILENAME="$PROJECT_NAME-changelog.txt"
     JIRA_CHANGELOG_FILENAME="$PROJECT_NAME-jira.txt"
 
-    
     if [ "$REPOSITORY_URL" != "null" ];
         then
             echo "[OK] $PROJECT_NAME($REPOSITORY_URL) -> $OLD_VERSION : $NEW_VERSION"
-            git clone "$REPOSITORY_URL $PROJECT_NAME" --quiet
+            git clone "$REPOSITORY_URL" "$PROJECT_NAME" --quiet
```

Comment:
These two quotes were the problem... :-(
the script was trying to checkout something like "git@bitbucket.org/qamine/codacy-worker.git engine" as a url, instead of the ssh url into the engine checkout folder.